packages/gallery/src/components/gallery/proGallery/galleryContainerExtraNew.js

```diff
@@ -536,6 +540,12 @@ export class GalleryContainer extends React.Component {
     }
 
     const displayShowMore = this.containerInfiniteGrowthDirection() === 'none';
+    const ssrDisableTransition =
```

Comment:
Let's discuss this again.
The hydrate should also no receive these transitions.
I think they should be removed completely

Reply:
i will remove it